client/extensions/woocommerce/state/sites/promotions/test/fixtures.js

```diff
@@ -0,0 +1,101 @@
+
+export const couponParams1 = { offset: 0, per_page: 5 };
+export const coupons1 = [
+	{
+		code: 'two',
+		amount: '2',
+		discount_type: 'fixed_cart',
+		date_created_gmt: '2017-09-07T12:50:50',
+		date_expires_gmt: '2017-10-20T12:50:50',
+	},
+	{
+		code: 'one',
+		amount: '1',
+		discount_type: 'fixed_cart',
+		date_created_gmt: '2017-09-06T13:54:50',
+		date_expires_gmt: '2017-10-15T13:54:50',
+	},
+	{
+		code: 'four',
+		amount: '4',
+		discount_type: 'percent',
+		date_created_gmt: '2017-09-09T09:50:50',
+		date_expires_gmt: '2017-10-28T09:50:50',
+	},
+	{
+		code: 'three',
+		amount: '3',
+		discount_type: 'percent',
+		date_created_gmt: '2017-09-08T10:50:50',
+		date_expires_gmt: '2017-10-25T10:50:50',
+	},
+	{
+		code: 'five',
+		amount: '5',
+		discount_type: 'percent',
+		date_created_gmt: '2017-09-10T10:50:50',
+		date_expires_gmt: undefined,
+	},
+];
+
+export const couponParams2 = { offset: 5, per_page: 5 };
+export const coupons2 = [
+	{
+		code: 'six',
+		amount: '6',
+		discount_type: 'fixed_cart',
+		date_created_gmt: '2017-09-11T12:50:50',
+		date_expires_gmt: undefined,
+	},
+	{
+		code: 'seven',
+		amount: '7',
+		discount_type: 'percent',
+		date_created_gmt: '2017-09-12T04:54:50',
+		date_expires_gmt: '2017-11-05T04:54:50',
+	},
+];
```

Comment:
Besides `fixed_cart` and `percent`, WooCommerce core also supports `fixed_product` coupons ( https://github.com/woocommerce/woocommerce/blob/71d0702523bcc365403dd3b84b945da5218d88b4/tests/unit-tests/coupon/functions.php#L20 ) - maybe add one of those to your test fixture/tests?